metron-platform/metron-storm-kafka/src/main/java/org/apache/metron/storm/kafka/flux/SimpleStormKafkaBuilder.java

```diff
@@ -165,13 +177,39 @@ public SimpleStormKafkaBuilder( Map<String, Object> kafkaProps
                                 , List<String> fieldsConfiguration
                                 )
   {
-    super( modifyKafkaProps(kafkaProps, zkQuorum)
-         , createStreams(fieldsConfiguration, topic)
-         , createTuplesBuilder(fieldsConfiguration, topic)
-         );
+    super( getBootstrapServers(zkQuorum, kafkaProps)
+         , createDeserializer(Optional.ofNullable((String)kafkaProps.get(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG)), DEFAULT_DESERIALIZER)
+         , createDeserializer(Optional.ofNullable((String)kafkaProps.get(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG)), DEFAULT_DESERIALIZER)
+         , topic
+    );
+    setProp(kafkaProps);
+    setRecordTranslator(new SpoutRecordTranslator<>(FieldsConfiguration.toList(fieldsConfiguration)));
     this.topic = topic;
   }
 
+  private static <T> Class<Deserializer<T>> createDeserializer( Optional<String> deserializerClass
```

Comment:
This is nitpicky, so don't feel obligated to fix it, but couldn't this method just take a (possibly null) String, and wrap it in the Optional itself?  It seems pointless to make the caller wrap it when it's just as easy for the function itself to handle null.